hbase-migration-tools/bigtable-hbase-replication/bigtable-hbase-2.x-replication/src/main/java/com/google/cloud/bigtable/hbase2_x/replication/configuration/CustomWALEntryConfig.java

```diff
@@ -0,0 +1,23 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.bigtable.hbase2_x.replication.configuration;
+
+/** Static class containing all the configuration keys and their descriptions. */
+public class CustomWALEntryConfig {
+  public static final String REPLICATION_WALENTRYFILTER_CONFIG_KEY =
```

Comment:
We should use the HBase constant here. We don't own this, it should be coming from the hbase-server jar installed on the cluster.

Reply:
Replacing with the constant from hbase-server jar.

Reply:
It is not present in HBase constant. It is a part of BaseReplicationEndpoint https://github.com/apache/hbase/blob/master/hbase-server/src/main/java/org/apache/hadoop/hbase/replication/BaseReplicationEndpoint.java#L41.